dev/PipsPager/PipsPager.cpp

```diff
@@ -116,7 +123,7 @@ void PipsPager::OnApplyTemplate()
 
 void PipsPager::RaiseSelectedIndexChanged()
 {
-    const auto args = winrt::make_self<PipsPagerSelectedIndexChangedEventArgs>(m_lastSelectedPageIndex, SelectedPageIndex());
+    const auto args = winrt::make_self<PipsPagerSelectedIndexChangedEventArgs>();
```

Comment:
Why did we remove the args from the event?

Reply:
During the API review we decided that the contents of the args were not necessary:
![image](https://user-images.githubusercontent.com/21356912/112063759-22dba880-8b1f-11eb-91f5-7d19a513cf52.png)
the current index can be extracted from the PipsPager itself and the previousIndex was considered not needed